Vagrantfile

```diff
@@ -26,28 +26,36 @@ end
 
 Vagrant.configure(2) do |config|
 
-    config.vm.box = "debian/jessie64"
+    # Prefer docker over virtualbox since it is listed first
+    # meaning that docker should run if no '--provider' is supplied
+    # when calling vagrant up
+    config.vm.provider "docker" do |d, configOverride|
+        d.build_dir = "."
+        d.pull = true
+        d.has_ssh = true
+
+        # Overrides for 'config' unique for docker
+        configOverride.ssh.username = "vagrant"
+        configOverride.ssh.password = "vagrant"
+    end
 
-    config.vm.provider "virtualbox" do |vb|
+    config.vm.provider "virtualbox" do |vb, configOverride|
```

Comment:
@jonte Good point. But I'm not saying that it's not wanted, I'm just saying that our support for it is likely to break unless we use it ourselves. Or if we actively test it etc ofc. But I'm not sure that would be prioritized unless we have actual users that wants it. Maybe we do?